src/gui/windows/multiplayer.zig

```diff
@@ -56,11 +56,18 @@ fn join(_: usize) void {
 		ipAddress = "";
 	}
 	if(connection) |_connection| {
+		if (ipAddressEntry.currentString.items.len == 0) {
+			std.log.err("IP address cannot be empty", .{});
+			gui.windowlist.notification.text = "IP address cannot be empty";
+			gui.openWindow("notification");
```

Comment:
What happens if the window is already open?

Reply:
> should every new issue get a new notification window

There can only be one instance of a given window, so no, it must reuse the same window.

> in case you wanna go with only one, the latest error text can be displayed on the notification window.

Yes that would be ideal.